packages/twenty-server/src/workspace/workspace-query-builder/factories/relation-field-alias.factory.ts

```diff
@@ -67,18 +76,13 @@ export class RelationFieldAliasFactory {
     );
     // Retrieve the referenced object metadata based on the relation direction
     // Mandatory to handle n+n relations
-    const referencedObjectMetadata =
```

Comment:
this was it! Killing the performance, I don't know how we missed it!

Reply:
I've tested it during the implementation, and the difference was not big on my side, maybe the amount of database schema was also related